pkg/seedadmission/extension_crds.go

```diff
@@ -156,3 +130,41 @@ func crdMustBeConsidered(logger *logrus.Entry, labels map[string]string) bool {
 
 	return true
 }
+
+func getRequestObject(ctx context.Context, c client.Client, request *admissionv1beta1.AdmissionRequest) (runtime.Object, error) {
```

Comment:
Can you move this function into a new, separate file now that it's no longer solely required by the extension CRD handler?